pyproject.toml

```diff
@@ -204,6 +205,9 @@ ignore = [
 [tool.ruff.lint.pydocstyle]
 convention = "pep257" # https://docs.astral.sh/ruff/settings/#lint_pydocstyle_convention
 
+[tool.lint.flake8-import-conventions.extend-aliases]
+"numpy.typing" = "npt"
```

Comment:
Here's a request to add this alias convention to the default aliases: https://github.com/astral-sh/ruff/issues/17028